src/tools/dev/scripts/bv_support/bv_mpich.sh

```diff
@@ -114,6 +114,11 @@ function build_mpich
     mpich_opts="--enable-shared"
     if [[ "$OPSYS" == "Darwin" ]]; then
         mpich_opts="${mpich_opts} --enable-two-level-namespace --enable-threads=single"
+	# For macOS (Catalina and Big Sur) IPv6 needs to be disabled
```

Comment:
This looks like an important change for macOS and again, feels to me like best if separated from changes related to *bv_cleanup*. Sorry if I seem like I am repeating this point too often.

Reply:
These probably could all be separate changes. But I would need to start over.